src/components/visualization/bar-chart/AxisSelector.tsx

```diff
@@ -0,0 +1,38 @@
+import { Dispatch, SetStateAction } from 'react';
+import { useTranslations } from 'next-intl';
+
+export default function AxisSelector({
+  axesMap,
+  setAxis,
+}: {
+  axesMap: Map<string, Map<string, boolean>>;
+  setAxis: Dispatch<SetStateAction<string>>;
+}) {
+  const t = useTranslations('AxisSelector');
+  if (axesMap.size < 2) {
+    return <div />;
+  }
+
+  const onAxisChange = (e: React.ChangeEvent<HTMLSelectElement>) => {
+    setAxis(e.target.value);
+  };
+
+  return (
+    <div>
```

Comment:
Please add a (bottom) margin between selector and visualization - probably `mb-2`